app/utils.py

```diff
@@ -58,6 +58,28 @@ def pagination_links(pagination, endpoint, args):
     return links
 
 
+def result_meta(total_count):
+    return {"total": total_count}
+
+
+def single_result_response(result_name, result, serialize_kwargs={}):
+    return jsonify(**{result_name: result.serialize(**serialize_kwargs)})
+
+
+def list_result_response(result_name, results_query, serialize_kwargs={}):
+    meta = result_meta(results_query.count())
+    serialized_results = [result.serialize(**serialize_kwargs) for result in results_query]
+    return jsonify(meta=meta, **{result_name: serialized_results})
+
+
+def paginated_result_response(result_name, results_query, page, per_page, endpoint, request_args, serialize_kwargs={}):
```

Comment:
This should probably have a docstring. In fact, they all probably should.

Reply:
Added, let me know what you think